Tests.SpecRun/TestData/Labels.feature.cs

```diff
@@ -115,12 +115,14 @@ public virtual void V1_2Api_CoreApiTest1()
         
         [TechTalk.SpecRun.ScenarioAttribute("[v1.2] [api.core.create] Api test 2", new string[] {
                 "api",
-                "create"}, SourceLine=10)]
+                "create",
+                "link:http://google.com"}, SourceLine=10)]
```

Comment:
can we use `example.org` links for examples?

Reply:
Sure. Fixed too.